src/index.tsx

```diff
@@ -1,10 +1,21 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
+import { ValidatorForm } from './components/ValidatorForm';
+
 import { Resource } from "./components/Resource";
 
-import { SelectOption } from './models/SelectOption';
-import { ProfileForm } from "./components/ProfileForm";
+const validatorElement = document.getElementById('validator');
+if (validatorElement) {
+  const basePath = validatorElement.getAttribute('data-base-path');
+  const profiles: Map<string, string[]> = JSON.parse(atob(validatorElement.getAttribute('data-profiles')));
+  ReactDOM.render(
+    <React.StrictMode>
```

Comment:
huh, TIL.  looks cool.

Reply:
Hmm I can see from this change that I think you're right that it should be a `Record` after the `JSON.parse`

Reply:
I think you meant to comment on the other topic 🙂

Reply:
I found out about StrictMode from the #react-js slack channel!